src/App.tsx

```diff
@@ -1,7 +1,30 @@
 function App() {
```

Comment:
The reason the css didnt show up is we need to import the css file.

Reply:
hey i added the import its working now thank you